app/queries/consumer_apps/rpush_app_query.rb

```diff
@@ -16,24 +17,24 @@ def initialize(app_bundle:, platform:)
 
     def call
       if consumer_app.android?
-        android_app = Rpush::Gcm::App.where(name: app_bundle).first
+        android_app = Rpush::Gcm::App.where(name: app_name).first
         android_app || recreate_android_app!
       elsif consumer_app.ios?
-        ios_app = Rpush::Apns2::App.where(name: app_bundle).first
+        ios_app = Rpush::Apns2::App.where(name: app_name).first
         ios_app || recreate_ios_app!
       end
     end
 
     private
 
-    attr_reader :app_bundle, :consumer_app
+    attr_reader :app_bundle, :platform, :app_name, :consumer_app
```

Comment:
Super nit, but I'd probably have added `platform` at the end to keep this sorted.